### ISSUE TITLE
Flaky unit test failure Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.CommentSelection.VisualBasicCommentSelectionTests.Comment1

### ISSUE THREAD

**Issue by mavasani** [dotnet/roslyn#25898]
See https://ci.dot.net/job/dotnet_roslyn/job/master/job/windows_debug_unit32_prtest/12813/testReport/junit/Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.CommentSelection/VisualBasicCommentSelectionTests/Comment1/

```
MESSAGE:
System.TimeoutException : Failed to clean up listeners in a timely manner.\r\n---- System.OperationCanceledException : The operation was canceled.
+++++++++++++++++++
STACK TRACE:
at Microsoft.CodeAnalysis.Test.Utilities.UseExportProviderAttribute.After(MethodInfo methodUnderTest) ----- Inner Stack Trace ----- at System.Threading.CancellationToken.ThrowOperationCanceledException() at System.Threading.CancellationToken.ThrowIfCancellationRequested() at Roslyn.Test.Utilities.TaskJoinExtensions.JoinUsingDispatcher(Task task, CancellationToken cancellationToken) at Microsoft.CodeAnalysis.Test.Utilities.UseExportProviderAttribute.After(MethodInfo methodUnderTest)
```

**Comment by mavasani** [comment]
Tagging @dotnet/roslyn-infrastructure @sharwell

**Comment by sharwell** [comment]
Should be fixed by #25892
